cordova-js-src/splashscreen.js

```diff
@@ -212,7 +215,7 @@ function exitFullScreen() {
 // Make title bg color match splashscreen bg color
 function colorizeTitleBar() {
     var appView = Windows.UI.ViewManagement.ApplicationView.getForCurrentView();
-    if (isWin10UWP) {
+    if (isWin10UWP && (typeof titleBgColor !== 'undefined')) {
```

Comment:
no nicer way to do this but then to check for undefined?
took some time to understand what is going on here, especially with the `titleInitialBgColor ` that depends on this one

Reply:
That's a reasonable point. Both `titleBgColor` and `titleInitialBgColor` are set in one place and used in one place. Adding a variable to track whether or not `bgColor` is transparent seemed like too much. Thoughts?

Reply:
First I thought you could just check if bgColor == 'transparent`, but that doesn't exist here. `isTitleBgTransparent` would be a solution and probably more readable, also makes the special case explicit.